yarn/cloudbuild.yaml

```diff
@@ -63,3 +78,5 @@ images:
 - 'gcr.io/$PROJECT_ID/yarn:node-8.4.0'
```

Comment:
Please add `:lts` to the list of pushed images.

Reply:
@bendory :lts tag is added to image list

Thank you!